frontend/public/components/factory/table.tsx

```diff
@@ -276,15 +276,52 @@ export type TableRowProps = {
   className?: string;
 };
 
+const BREAKPOINT_SM = 576;
+const BREAKPOINT_MD = 768;
+const BREAKPOINT_LG = 992;
+const BREAKPOINT_XL = 1200;
+const BREAKPOINT_XXL = 1400;
+const MAX_COL_SM = 2;
+const MAX_COL_MD = 4;
+const MAX_COL_LG = 4;
+const MAX_COL_XL = 6;
+const MAX_COL_XXL = 8;
+
+const isColumnVisible = (columnID: string, columns: Set<string> = new Set()) => {
```

Comment:
Is this reevaluated when the window is resized?

Reply:
@spadgett this is reevaluated when the window is resized. The window resize was already causing a re render based on the resize listener.